config/default.yml

```diff
@@ -4880,6 +4880,11 @@ Style/RedundantRegexpEscape:
   Enabled: true
   VersionAdded: '0.85'
 
+Style/RedundantRegexpQuantifiers:
+  Description: 'Checks for redundant quantifiers in Regexps.'
+  Enabled: true
```

Comment:
or `Enabled: pending`? not sure what to choose

Reply:
All new cops should be marked as pending.